src/CMakeLists.txt

```diff
@@ -229,3 +177,22 @@ target_include_directories(parthenon PUBLIC
 )
 
 lint_target(parthenon)
+
+# Generate install targets for make if CMAKE_INSTALL_PREFIX has been provided
+if(NOT CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
```

Comment:
I don't understand why you're checking for this - why shouldn't it use the default install path if that's what the user wants?

Reply:
You're right, this is gone -- I was too focused on my own use case. I don't think this `if` statement is actually needed at all anyway; if the user doesn't want to install, they just don't call `cmake --install` and they're fine as I understand it.